test/Conversion/tritongpu_to_llvm.mlir

```diff
@@ -606,40 +675,76 @@ module attributes {"triton_gpu.num-warps" = 4 : i32} {
+    // PTX: llvm.mlir.constant(0 : i32) : i32
+    // PTX: llvm.add
+    // PTX: llvm.inline_asm
+    // PTX-SAME: cp.async.ca.shared.global [ ${{.*}} + 0 ], [ ${{.*}} + 0 ], 0x4, 0x4
+    // PTX: llvm.mlir.constant(0 : i32) : i32
+    // PTX: llvm.add
+    // PTX: llvm.inline_asm
+    // PTX-SAME: cp.async.ca.shared.global [ ${{.*}} + 0 ], [ ${{.*}} + 0 ], 0x4, 0x4
+    // PTX: llvm.mlir.constant(0 : i32) : i32
+    // PTX: llvm.add
+    // PTX: llvm.inline_asm
+    // PTX-SAME: cp.async.ca.shared.global [ ${{.*}} + 0 ], [ ${{.*}} + 0 ], 0x4, 0x4
+    // PTX: llvm.mlir.constant(16 : i32) : i32
+    // PTX: llvm.add
+    // PTX: llvm.inline_asm
+    // PTX-SAME: cp.async.ca.shared.global [ ${{.*}} + 0 ], [ ${{.*}} + 0 ], 0x4, 0x4
+    // PTX: llvm.mlir.constant(16 : i32) : i32
+    // PTX: llvm.add
+    // PTX: llvm.inline_asm
+    // PTX-SAME: cp.async.ca.shared.global [ ${{.*}} + 0 ], [ ${{.*}} + 0 ], 0x4, 0x4
+    // PTX: llvm.mlir.constant(16 : i32) : i32
+    // PTX: llvm.add
+    // PTX: llvm.inline_asm
+    // PTX-SAME: cp.async.ca.shared.global [ ${{.*}} + 0 ], [ ${{.*}} + 0 ], 0x4, 0x4
+    // PTX: llvm.mlir.constant(16 : i32) : i32
+    // PTX: llvm.add
+    // PTX: llvm.inline_asm
+    // PTX-SAME: cp.async.ca.shared.global [ ${{.*}} + 0 ], [ ${{.*}} + 0 ], 0x4, 0x4
+    // PTX: llvm.inline_asm
+    // PTX-SAME: cp.async.commit_group
+
+    // TODO Check this code correctness
+    // GCN: llvm.mlir.constant(0 : i32) : i32
+    // GCN: llvm.add
+    // GCN: llvm.mlir.constant(0 : i32) : i32
+    // GCN: llvm.add
+    // GCN: llvm.mlir.constant(0 : i32) : i32
+    // GCN: llvm.add
+    // GCN: llvm.mlir.constant(0 : i32) : i32
+    // GCN: llvm.add
+    // GCN: llvm.mlir.constant(16 : i32) : i32
+    // GCN: llvm.add
+    // GCN: llvm.mlir.constant(16 : i32) : i32
+    // GCN: llvm.add
+    // GCN: llvm.mlir.constant(16 : i32) : i32
+    // GCN: llvm.add
+    // GCN: llvm.mlir.constant(16 : i32) : i32
+    // GCN: llvm.add
+    // GCN: llvm.inline_asm
+    // GCN-SAME: cp.async.ca.shared.global [ ${{.*}} + 0 ], [ ${{.*}} + 0 ], 0x4, 0x4
+    // GCN: llvm.inline_asm
+    // GCN-SAME: cp.async.ca.shared.global [ ${{.*}} + 0 ], [ ${{.*}} + 0 ], 0x4, 0x4
+    // GCN: llvm.inline_asm
+    // GCN-SAME: cp.async.ca.shared.global [ ${{.*}} + 0 ], [ ${{.*}} + 0 ], 0x4, 0x4
+    // GCN: llvm.inline_asm
+    // GCN-SAME: cp.async.ca.shared.global [ ${{.*}} + 0 ], [ ${{.*}} + 0 ], 0x4, 0x4
+    // GCN: llvm.inline_asm
+    // GCN-SAME: cp.async.ca.shared.global [ ${{.*}} + 0 ], [ ${{.*}} + 0 ], 0x4, 0x4
+    // GCN: llvm.inline_asm
+    // GCN-SAME: cp.async.ca.shared.global [ ${{.*}} + 0 ], [ ${{.*}} + 0 ], 0x4, 0x4
+    // GCN: llvm.inline_asm
+    // GCN-SAME: cp.async.ca.shared.global [ ${{.*}} + 0 ], [ ${{.*}} + 0 ], 0x4, 0x4
+    // GCN: llvm.inline_asm
+    // GCN-SAME: cp.async.ca.shared.global [ ${{.*}} + 0 ], [ ${{.*}} + 0 ], 0x4, 0x4
+    // GCN: llvm.inline_asm
+    // GCN-SAME: cp.async.commit_group
```

Comment:
Actually these inline asm (e.g. `cp.async.ca.shared.global`) are NV instrinsics, so I suggest to leave here `CHECK` lines and add perhaps a "TODO".
At the moment there is no sense to distinguish PTX and AMD targets, since the latter generates not working asm inlines.

Reply:
I agree that GCN tags are not needed in this case, but I think it make sense to rename `CHECK` to `PTX`, otherwise this test will fail.

Reply:
After discussion with Daniil and some investigation I found that these checks in our fork are outdated.
So I've updated them according upstream and revert them to CHECK tag.